src/FSharpx.Collections/Collections.fs

```diff
@@ -230,6 +230,13 @@ module Seq =
               notFirst := true
       } 
 
+    /// The catOptions function takes a list of Options and returns a seq of all the Some values.
+    let catOptions (xs:seq<Option<'a>>) =
+        seq {
```

Comment:
mmm tbh I don't really agree gere. But maybe we find others that can join discussion